sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -505,7 +505,12 @@ class SparkSqlAstBuilder extends AstBuilder {
           } else {
             None
           }
-          (Seq.empty, Option(name), props.toSeq, recordHandler)
+          val finalProps = if (!props.contains("field.delim")) {
```

Comment:
is this case sensitive?

Reply:
> is this case sensitive?

Yes, case sensitive
![image](https://user-images.githubusercontent.com/46485123/103199858-d4a37880-4926-11eb-9d58-dae7b809e2c7.png)